src/pages/categories/form/CategoryFormFields.tsx

```diff
@@ -0,0 +1,113 @@
+import i18n from '@dhis2/d2-i18n'
+import { CheckboxFieldFF, RadioFieldFF } from '@dhis2/ui'
+import React from 'react'
+import { Field } from 'react-final-form'
+import {
+    CustomAttributesSection,
+    DefaultIdentifiableFields,
+    StandardFormField,
+    StandardFormSection,
+    StandardFormSectionTitle,
+    StandardFormSectionDescription,
+    DescriptionField,
+    HorizontalFieldGroup,
+    ModelTransferField,
+} from '../../../components'
+import { SECTIONS_MAP } from '../../../lib'
+
+export const CategoryFormFields = () => {
+    const section = SECTIONS_MAP.category
+
+    return (
+        <>
+            <StandardFormSection>
+                <StandardFormSectionTitle>
+                    {i18n.t('Basic information')}
+                </StandardFormSectionTitle>
+                <StandardFormSectionDescription>
+                    {i18n.t('Set up the basic information for this category.')}
+                </StandardFormSectionDescription>
+                <DefaultIdentifiableFields />
+                <DescriptionField
+                    schemaSection={section}
+                    helpText={i18n.t('Explain the purpose of this category.')}
+                />
+            </StandardFormSection>
+
+            <StandardFormSection>
+                <StandardFormSectionTitle>
+                    {i18n.t('Data configuration')}
+                </StandardFormSectionTitle>
+                <StandardFormSectionDescription>
+                    {i18n.t(
+                        'Choose how this category will be used to capture and analyze'
+                    )}
+                </StandardFormSectionDescription>
+                <StandardFormField>
```

Comment:
The only thing `StandardFormField` does currently is adding margins. I think I would prefer adding `gap` to `StandardFormSection` instead. That would cleanup quite a lot in the form-fields...